### ISSUE TITLE
MySolarSystemControls cleanup

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/my-solar-system#196]
Reorganization and cleanup of MySolarSystemControls should be done before PhET-iO instrumentation.

In MySolarSystemControls, these `create` functions are adding nothing.  The UI checkboxes should simply be instantiated directly. 

```ts
      children: [
        ...createOrbitalInformationCheckboxes( model, providedOptions.tandem ),
        new HSeparator( SolarSystemCommonConstants.HSEPARATOR_OPTIONS ),
        ...createArrowsVisibilityCheckboxes( model, providedOptions.tandem ),
        new GravityScaleSlider( model.forceScaleProperty, model.gravityVisibleProperty ),
        new HSeparator( SolarSystemCommonConstants.HSEPARATOR_OPTIONS ),
        ...createVisibilityInformationCheckboxes( model, providedOptions.tandem )
      ],
```

In MySolarSystemScreenView, it's incorrectly assigned to `controlPanel`, which is later wrapped in a Panel. So I'm planning to rename it MySolarSystemPanel, and make it `extend Panel`.

```ts
    const controlPanel = new MySolarSystemControls( model, this.topLayer, {
      tandem: providedOptions.tandem.createTandem( 'controlPanel' )
    } );

    this.topRightControlBox = new VBox( {
...
      children: [
...
        new Panel( controlPanel, SolarSystemCommonConstants.CONTROL_PANEL_OPTIONS )
      ]
    } );
```

**Comment by pixelzoom** [comment]
I got a good start on this.  But cleaning up `createArrowsVisibilityCheckboxes` and `createVisibilityInformationCheckboxes` requires changed in solar-system-common.  So I'll need to put this on hold until Kepler's Laws 1.1 is published.

**Comment by pixelzoom** [comment]
@AgustinVallejo and I discussed the name of this panel, and we like VisibilityControlPanel.

**Comment by pixelzoom** [comment]
Functions `createOrbitalInformationCheckboxes`, `createArrowsVisibilityCheckboxes`, and `createVisibilityInformationCheckboxes` no longer exist.  Instead, checkboxes are created using static methods provided by these classes:

* SolarSystemCommonCheckbox creates checkboxes that are shared by both sims
* MySolarSystemCheckbox extends SolarSystemCommonCheckbox, and creates checkboxes that are specific to my-solar-system
* KeplersLawsCheckbox extends SolarSystemCommonCheckbox, and creates checkboxes that are specific to keplers-laws

Over to @AgustinVallejo for review. Close if OK.

**Comment by AgustinVallejo** [comment]
Making these functions static is definitely a step in the right direction, thanks. Closing.
